waspc/data/Generator/templates/sdk/package.json

```diff
@@ -141,7 +141,11 @@
       {=!  Used by users, documented. =}
       "./webSocket": "./dist/webSocket/index.js",
       {=!  Used by our code, uncodumented (but accessible) for users. =}
-      "./webSocket/WebSocketProvider": "./dist/webSocket/WebSocketProvider.jsx"
+      "./webSocket/WebSocketProvider": "./dist/webSocket/WebSocketProvider.jsx",
+
+      {=! ================= NEW API HERE =================== =}
+      {=! Only public exports =}
+      "./server/api": "./dist/server/api/index.js"
```

Comment:
Let's put all the new routes here, so it is clear which ones are public and which ones are not.